app.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+const sequelize = require('./config/database');
+const routes = require("./routes/index");
+
+const app = express();
+const PORT = process.env.PORT || 3001;
+
+app.use(cors());
+app.use(bodyParser.json());
+app.use('/api', routes);
+
+app.use((err, req, res, next) => {
+  console.error(err); // Log the error for debugging
+  res.status(500).json({ message: 'Internal Server Error' });
+});
+
+sequelize
+  .sync()
+  .then(() => {
+    console.log('Connected to the database');
+    app.listen(PORT, () => console.log(`Server is running on port ${PORT}`));
+  })
+  .catch((err) => console.error('Database connection failed:', err));
```

Comment:
I think so, it's better that we should call sync inside `models` import file and I think so listen isn't `dependent` on `sync`